lib/seattleflu/id3c/cli/command/etl/redcap_det_uw_reopening.py

```diff
@@ -840,3 +842,41 @@ def get_date_from_repeat_instance(instance_id: int) -> str:
     Returns the date as a string because that is how it is used.
     """
     return (STUDY_START_DATE + relativedelta(days=(instance_id -1))).strftime('%Y-%m-%d')
+
+
+def insert_range_of_synthetic_enrollment_dets(start_record_id: int, end_record_id: int,
+        generated_by: str) -> None:
+    """
+    Inserts synthetic DET records into the `receiving.redcap_det` table to mimic
+    changes to the Enrollment Questionnaire for the range of record IDs (inclusive)
+    specified.
+    Requires the `REDCAP_API_TOKEN_{REDCap URL}_{REDCap project ID}` environment variable be set.
+    """
+    project = Project(REDCAP_URL, PROJECTS[0].id, dry_run = False)
+
+    record = {
+        "project" : project,
+        "enrollment_questionnaire_complete" : "", # Don't set a value
+        "redcap_event_name" : ENROLLMENT_EVENT_NAME,
+        }
+
+    current_record_id = start_record_id
+    dets = []
+
+    while current_record_id <= end_record_id:
```

Comment:
I'm typically used to seeing `while` statements when we don't know when the loop will exit. In this case, we know in advance when we're finished (e.g. when `current_record_id` > `end_record_id`). In this case, I think it's better to use a `for` loop. e.g.

`for record_id in range(current_record_id, end_record_id + 1):`